emperor/support_files/templates/logic-template.html

```diff
@@ -97,21 +97,29 @@
 
   var div = $('#{{ plot_id }}');
 
-  var ids = {{ coords_ids }};
-  var coords = {{ coords }};
-  var pct_var = {{ pct_var | map('float') | list }};
-  var md_headers = {{ md_headers}};
-  var metadata = {{  metadata }};
-  var axesNames = {{ axes_names }};
+  var data = {
+    // coordinates information
+    'sample_ids': {{ coords_ids | tojson }},
```

Comment:
Yeah, they call them filters in the JINJA world http://jinja.pocoo.org/docs/2.9/templates/#filters